packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -0,0 +1,93 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-footnotes');
+const { overrideArgs } = storybookHelpers('sd-footnotes');
+const { generateTemplate } = storybookTemplate('sd-footnotes');
+
+/**
+ * A footnote contains additional information/sources related to the content and usually appears at the bottom of a page or below the content it refers to.
+ */
+
+export default {
+  title: 'Styles/sd-footnotes',
+  component: 'sd-footnotes',
+  parameters: {
+    ...parameters,
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/DeZ6iJggYfpSXHcEexNeYs/Footnote?type=design&node-id=0-1&mode=design&t=7vjD8Qq8iKSnKn6a-0'
+    }
+  },
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value:
+        '<li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</li><li>Sed diam nonumy eirmod tempor invidunt ut labore.</li><li>Dolore magna aliq erat, sed diam voluptua.</li>'
+    }
+  ]),
+  argTypes,
+  decorators: [
+    (story: () => typeof html) => html`
+      <style>
+        td.template {
+          text-align: left;
+        }
+      </style>
+      ${story()}
+    `
+  ]
+};
+
+/**
+ * Default: This shows sd-footnote in its default state, used with a `ul` element.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      options: { templateContent: '<div class="sd-container"><ul class="%CLASSES%">%SLOT%</ul></div?' },
+      args
+    });
+  }
+};
+
+/**
+ * Use the class with a `ol` element to display the footnotes as a numbered list.
```

Comment:
@Vahid1919 what do you think about switching the "default" here? A footnote list without numbers (ul) doesn't really match the expectations of a footnote list (IMO).
maybe even with only one lingle list item as you always should add numbers if you have more. 

@coraliefeil what do you think? is there even a use case für a footnote list without numbers? 

If so, I would like to see the case specified as best as possible.